components/alert/alert.css

```diff
@@ -0,0 +1,56 @@
+/* alert */
+.section-alert {
+    display: flex;
+    flex-direction: column;
+    gap: 10px; 
+}
+.alert{
+    border-radius: 0.3rem;
+    font-size:2rem;
+    height:4rem;
+    width:46rem;
+    padding:15px 5px 5px 15px;
+    margin: 5px; 
+}
+.alert-primary {
+    background-color: #a2d6f9;
```

Comment:
Use variables to define the colors in stylesheets.

Reply:
Okay I will.